oer-api/app/controllers/oer/NtToEs.java

```
@@ -112,7 +112,7 @@ private static void process(Map<String, StringBuilder> map) {
 		for (Entry<String, StringBuilder> e : map.entrySet()) {
 			try {
 				String id = e.getKey().split("\\.")[0];
-				indexData(idMap.isEmpty() ? id : idMap.get(id),
+				indexData(idMap.isEmpty() || idMap.get(id)==null ? id : idMap.get(id),
```

I checked for `isEmpty` only here on purpose: I was thinking we should either use all the file names or just mapped IDs. If an ID is missing I'd prefer an error over silently using the file name for a single entry. I believe that leads to weird issues later (no error during indexing, but missing/inconsistent IDs in the data). Thoughts?